src/io/readWriteImage.ts

```diff
@@ -34,7 +34,8 @@ export const writeImage = async (format: string, image: vtkImageData) => {
     }
```

Comment:
Can remove this workaround now.

Remove writeImage direction matrix workaround after upgrading @itk-wasm/image-io #538

Reply:
Cool, thanks for the heads up!